my_store/blog/dashboard/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
```

Comment:
See how each module should order by this document https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/